scripts/entrypoint.sh

```diff
@@ -28,7 +28,10 @@ function wait_for_db() {
 wait_for_db
 
 if $LOAD_FIXTURES == "true"; then
-    python manage.py loaddata fake_data
+    python make_fake_fixtures.py 100 1000 1000 > /tmp/fake_agagd_data.json
+    python manage.py loaddata /tmp/fake_agagd_data.json
+else
+    rm -f make_fake_fixtures.py /tmp/fake_agagd_data.json
```

Comment:
Not sure we really need this else clause. `/tmp/fake_agagd_data.json` shouldn't exist in the first place if we started the container with `LOAD_FIXTURES` set to false. And I *think* you can't trivially change the environment variables passed into a container after the container is created; the whole container (and its ephemeral volumes) will get recreated from the base image.

And I'm not sure it's super *useful* to selectively delete the make_fake_fixtures script. It just hangs out and takes up a couple KB, surely...

Reply:
Sounds good. It is now removed in the current version.